packages/flutter/lib/src/cupertino/theme.dart

```diff
@@ -473,6 +497,11 @@ class NoDefaultCupertinoThemeData {
   /// Defaults to [CupertinoColors.systemBackground].
   final Color? scaffoldBackgroundColor;
 
+  /// Color of the selection handle.
+  ///
+  /// Defaults to [CupertinoColors.activeBlue].
```

Comment:
Actually systemBlue I think?

Reply:
Thankss! I will fix that and the analyzer failures

Reply:
Done!